content/en/docs/how-tos/overriding-failing-ci-jobs.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Overriding Failing CI Jobs"
+description: How and when to override a failing CI job in order to allow a pullrequest to merge
+---
+This document talks about how to bypass a failing CI job so that a PR can merge, and when it is appropriate to do so.
+
+## How to override
+
+Putting `/override $ci-job-name` (e.g. `/override ci/prow/e2e-aws-single-node-serial`, note that the `ci/prow/` prefix is necessary)
+in a PR comment will cause tide to ignore the test failure and process the PR for merging if all other requirements are met.  In order
+to merge, a PR must have been tested (or overridden) on top of the current head of the branch it is merging into.  This means that even
+if you override a failing job, if another PR has merged into the repository in the meantime, the PR will still be retested(and
+potentially fail) during the merge attempt.  This means that `/override` is more likely to be helpful in a low traffic repository where
+nothing else is likely to have merged in the meantime.
+
+## Who CAN override
+
+Permission to use `/override` is determined by the approvers in the top level OWNERS file in the repository.
```

Comment:
https://prow.k8s.io/command-help says

"Repo administrators."

Reply:
my understanding is it's both, at least for us, but most people are not repo admins.

see: https://github.com/openshift/release/blob/6a62db67e37797814f900d32d71eab7787aa3582/core-services/prow/02_config/_plugins.yaml#L1887-L1888

Reply:
I also noticed that I was able to issue an `/override` when I wasn't a repo admin, FYI

Reply:
OhOh.
I saw that configuration before. Totally forgot that. My bad.